[2025-02-05T13:56:01.089+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-05T13:56:01.109+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_que_faire_a_paris.transform_data manual__2025-02-05T13:55:54.423054+00:00 [queued]>
[2025-02-05T13:56:01.122+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_que_faire_a_paris.transform_data manual__2025-02-05T13:55:54.423054+00:00 [queued]>
[2025-02-05T13:56:01.124+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-05T13:56:01.143+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): transform_data> on 2025-02-05 13:55:54.423054+00:00
[2025-02-05T13:56:01.150+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=91) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-05T13:56:01.152+0000] {standard_task_runner.py:72} INFO - Started process 93 to run task
[2025-02-05T13:56:01.151+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_que_faire_a_paris', 'transform_data', 'manual__2025-02-05T13:55:54.423054+00:00', '--job-id', '72', '--raw', '--subdir', 'DAGS_FOLDER/dag_que_faire_a_paris.py', '--cfg-path', '/tmp/tmpn6p4doih']
[2025-02-05T13:56:01.153+0000] {standard_task_runner.py:105} INFO - Job 72: Subtask transform_data
[2025-02-05T13:56:01.184+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_que_faire_a_paris.transform_data manual__2025-02-05T13:55:54.423054+00:00 [running]> on host 878f1ba0d333
[2025-02-05T13:56:01.353+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_que_faire_a_paris' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-05T13:55:54.423054+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-05T13:55:54.423054+00:00'
[2025-02-05T13:56:01.355+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-05T13:56:01.493+0000] {dag_que_faire_a_paris.py:137} INFO - âœ… Transformation completed successfully!
[2025-02-05T13:56:01.494+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-05T13:56:01.499+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-05T13:56:01.500+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_que_faire_a_paris, task_id=transform_data, run_id=manual__2025-02-05T13:55:54.423054+00:00, execution_date=20250205T135554, start_date=20250205T135601, end_date=20250205T135601
[2025-02-05T13:56:01.526+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-05T13:56:01.545+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-05T13:56:01.548+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
